apidoc/Titanium/UI/WebView.yml

```diff
@@ -541,6 +544,7 @@ events:
     properties:
       - name: url
         summary: The URL of the web document that is stopped.
+        type: String
```

Comment:
Same as above, There's also a hard-coded `message` String property. I think it always returns `"Webview did not load blacklisted url."`